src/renderers/WebGLRenderer.js

```diff
@@ -1862,6 +1867,10 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 				refreshUniformsPhong( m_uniforms, material );
 
+			} else if ( material instanceof THREE.MeshPhysicalMaterial ) {
```

Comment:
That was intentional.

Checking `instanceof THREE.MeshPhysicalMaterial` _first_ is required because we want to call a different method ( `refreshUniformsPhysical()` ), and we are in an `else if` construct.